shared/dataflow/codeql/dataflow/VariableCapture.qll

```diff
@@ -562,7 +583,7 @@ module Flow<InputSig Input> implements OutputSig<Input> {
 
   /**
    * Holds if `access` is a reference to `ce` evaluated in the `i`th node of `bb`.
-   * The reference is restricted to be in the same callable as `ce` as a
+   * The reference is restricted to be nested within the same callable as `ce` as a
```

Comment:
With this update, I'd like this predicate renamed from `localClosureAccess` to `localOrNestedClosureAccess`. Or something like this - the "local" prefix certainly is no longer true.